### DISCUSSION TITLE
What is the correct usage for GotoAsync in MAUI

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/maui#3387] [Q&A]
I see that in Weather21 Shell.Current.GotoAsync is used (and somehow doesn't generate an error. I get warnings using this and am wondering about it's correct usage?

**Comment by hartez** [comment]
> I see that in Weather21 Shell.Current.GotoAsync is used (and somehow doesn't generate an error. I get warnings using this and am wondering about it's correct usage?

Can you tell us what warnings you are seeing and how you are using it?

**Reply by bmrnjw** [reply]
I see the same issue navigating to a login page
